coc.html

```diff
@@ -43,40 +43,57 @@
   <!-- <div id="preloader">
     <div id="load"></div>
   </div> -->
-
-  <nav class="navbar navbar-custom navbar-fixed-top" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
-
-      <a class="navbar-brand" rel="home" href="https://in.pycon.org/2018/" title="PyCon India 2018">
+  <nav class="navbar navbar-custom navbar-fixed-top">
+    <div class="container">
+      <a class="navbar-brand index-logo" rel="home" href="https://in.pycon.org/2018/" title="PyCon India 2018">
         <img src="./img/banner-logo.png" class="logo" alt="banner-logo" style="width: 150px; display: inline-block;">
       </a>
       <div class="navbar-header">
-        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
-          <span class="sr-only">Toggle navigation</span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-        </button>
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+      <span class="sr-only">Toggle navigation</span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      </button>
       </div>
 
-      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-        <ul class="nav nav-justified">
-          <li><a href="index.html#home">Home</a></li>
-          <li><a href="index.html#journey">Journey</a></li>
-          <li><a href="index.html#speakers">Speakers</a></li>
-          <!-- <li><a href="#conference">Schedule</a></li> -->
+     <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+        <ul class="nav navbar-nav">
+          <li class="active"><a href="#home">Home</a></li>
```

Comment:
I think you didn't get what I said. This doesn't needs to be active. The links in href are incorrect. It should be index.html#home instead of just #home. Fix this everywhere.

Reply:
@Rohithgilla12 CoC is still left. Can you check the diff and proceed. That way you won't miss any file.